token/delegation/delegation.go

```diff
@@ -44,16 +46,15 @@ type Token struct {
 	expiration *time.Time
 }
 
-// New creates a validated Token from the provided parameters and options.
+// New creates a validated delegation Token from the provided parameters and options.
+// This is typically used to delegate a given power to another agent.
 //
-// When creating a delegated token, the Issuer's (iss) DID is assembled
-// using the public key associated with the private key sent as the first
-// parameter.
-func New(iss, aud did.DID, cmd command.Command, pol policy.Policy, opts ...Option) (*Token, error) {
+// You can read it as "(issuer) allows (audience) to perform (cmd+pol) on (subject)".
```

Comment:
Glad you included these - I was thinking overnight that we could also state these in "user story" form.  Perhaps something like "As an Issuer, I want to allow my Audience to Invoke Command+Policy on Subject".  I tried to be consistent with https://github.com/ucan-wg/spec?tab=readme-ov-file#roles.